drizzle-orm/src/pg-core/policy.ts

```diff
@@ -0,0 +1,55 @@
+import { entityKind } from '~/entity.ts';
+import type { AnyPgTable } from '~/pg-core/table.ts';
+import type { SQL } from '~/sql/sql.ts';
+import type { AnyPgRole } from './role.ts';
+
+export type PgPolicyFor = 'select' | 'insert' | 'update' | 'delete' | 'all';
+
+export type PgPolicyTo = 'public' | 'current_role' | 'current_user' | 'session_user';
```

Comment:
For Supabase, we have other roles like `authenticated`, etc.

Maybe the type could be `'public' | 'current_role' | 'current_user' | 'session_user' | (string & {})`

Reply:
Oh you handle that with `AnyPgRole`, sorry.
But, how can we use existing roles not created by us in the schema?

Reply:
`.existing()` like for views?

Reply:
Maybe you could inline the pgRole in the config like this:
```ts
{ to: [pgRole("my_existing_role")] }
```
Because all it needs is the name

Reply:
Sounds good